app/views/layouts/application.html.erb

```diff
@@ -13,7 +13,7 @@
     
   </head>
 
-  <body class="h-full font-sans">
+  <body class="h-full font-sans" data-controller="action-bar" data-action-bar-batch-destroy-url-value="<%= batch_destroy_tasks_path -%>">
```

Comment:
You mentioned sharing the routing between JS and Rails. I think this is pretty standard in Stimulus land: passing it in via data attributes.

Reply:
Though I guess it isn't needed anymore, since `button_to` is handling it now. Heh.